### PR TITLE
add ajax api to reset schedule's submitter(DO NOT MERGE)

### PR DESCRIPTION
**Pull request by chengren311**
1. add ajax api to reset schedule's submitter, e.x: http://localhost:8081/schedule?ajax=setScheduleSubmitter&scheduleId=2&submitter=someone
2. some refactoring by intellij save plugin
3. TriggerManager needs to persist updated trigger when updating trigger

### CODE REVIEW

**Comment by HappyRay** [comment]
Why is this API needed? It seems odd to have the API to change the submitter without changing the schedule.

**Comment by kunkun-tang** [comment]
@chengren311 It would be rational to check in this code to another branch, rather than master.

**Comment by chengren311** [comment]
@HappyRay we can talk offline. In a nutshell, trigger data is also kept in memory, it's very tricky to backfill to db only. Providing API is safest and most convenient way I can think of.

**Comment by chengren311** [comment]
@kunkun-tang agree, i will add DO NOT MERGE in the title of this PR.

File: azkaban-common/src/main/java/azkaban/scheduler/Schedule.java [hunk 1]
```diff
@@ -244,6 +244,10 @@ public String getSubmitUser() {
     return this.submitUser;
   }
 
+  public void setSubmitUser(String submitUser) {
+    this.submitUser = submitUser;
```
Comment (shuzhang1989) [inline azkaban-common/src/main/java/azkaban/scheduler/Schedule.java]: builder pattern?

File: azkaban-common/src/main/java/azkaban/trigger/TriggerManager.java [hunk 2]
```diff
@@ -329,7 +336,8 @@ private void checkAllTriggers() throws TriggerManagerException {
              * as triggerCondition do. As a consequence, we need to figure out a way to distinguish
              * the previous ExpireCondition and this commit's ExpireCondition.
              */
-            if (t.getExpireCondition().getExpression().contains("EndTimeChecker") && t.expireConditionMet()) {
+            if (t.getExpireCondition().getExpression().contains("EndTimeChecker") && t
+                .expireConditionMet()) {
```
Comment (shuzhang1989) [inline azkaban-common/src/main/java/azkaban/trigger/TriggerManager.java]: format

**Pull request closed without merging**